### PR TITLE
Overhead/make tests pass

### PR DESCRIPTION
**Pull request by cmac1000**
I'm doing some mildly weird things, particularly in `its/loaders/file_system.py`, to make tests pass - please take a look and set me straight (or if this is fine, let me know)

### CODE REVIEW

File: its/loaders/file_system.py [hunk 1]
```diff
@@ -4,7 +4,9 @@
 from ..errors import NotFoundError
 from io import BytesIO
 from ..settings import NAMESPACES
-import re
+
+from pathlib import PosixPath
+
 
 class FileSystemLoader(BaseLoader):
```
Comment (cmac1000) [inline its/loaders/file_system.py]: is FileSystemLoader intended to be used anywhere other than for local/testing purposes? If not, I'm less nervous about making these changes :)

**Comment by cmac1000** [comment]
@wamberg do we need to do any particular rain dance to kick off circleCI?

File: its/loaders/file_system.py (hunk 1 above)
Comment (jamesturk) [reply its/loaders/file_system.py]: filesystem loader was intended to be used w/ the overlay image, which we'd presumably have on the FS somewhere 

(taking a look at this now)

**Comment by jamesturk** [comment]
it looks OK to me, I actually wonder if the better solution if we're running in ECS is to switch the FileSystemLoader usage to something that keeps the image in memory (since it is meant to be used for overlay and nothing else) that wasn't a practical option w/ Lambda but it should be w/ ECS (load from S3 once-per-process should be fine since a process will probably last long enough to serve 1000s of requests)

**Comment by cmac1000** [comment]
closing in favor of #75

**Pull request closed without merging**